mail_show_follower/models/mail_mail.py

```diff
@@ -60,96 +60,34 @@ def remove_p(markup_txt):
-                            )
-                        )
-                    )
-                    if partners_len > 1:
-                        # get partners
-                        cc_internal = True
-                        # else get company in object
-                        if hasattr(obj, "company_id") and obj.company_id:
-                            cc_internal = obj.company_id.show_internal_users_cc
-                        # get company in user
-                        elif mail.env and mail.env.user and mail.env.user.company_id:
-                            cc_internal = (
-                                self.env.user.company_id.show_internal_users_cc
-                            )
-                        if cc_internal:
-                            partners = partners_obj.filtered(
-                                lambda x: x.id not in user_partner_ids
-                                and (
-                                    not x.user_ids
-                                    or any(x.mapped("user_ids.show_in_cc"))
-                                )
-                            )
-                        else:
-                            partners = partners_obj.filtered(
-                                lambda x: x.id not in user_partner_ids
-                                and (
-                                    not x.user_ids
-                                    or group_portal in x.user_ids.mapped("groups_id")
-                                )
-                            )
-                        partners = partners.filtered(
-                            lambda x: not x.user_ids
-                            or x.user_ids  # otherwise, email is not sent
-                            and "email" in x.user_ids.mapped("notification_type")
-                        )
-                        # set proper lang for recipients
-                        langs = list(
-                            filter(
-                                bool,
-                                mail.mapped("recipient_ids.lang")
-                                + [
-                                    mail.author_id.lang,
-                                    self.env.company.partner_id.lang,
-                                ],
-                            )
-                        )
-                        # get show follower text
-                        final_cc = mail.with_context(
-                            lang=langs and langs[0]
-                        )._build_cc_text(partners)
-                        # it is saved in the body_html field so that it does
-                        # not appear in the odoo log
-                        mail.body_html = final_cc + mail.body_html
+
+        for mail in self:
+            if not (mail.model and mail.res_id and group_portal):
+                continue
+            recipients = mail.recipient_ids | mail.notified_partner_ids
```

Comment:
suggestion: as said in https://github.com/OCA/social/pull/855#issuecomment-1575081147, one of the failures this module has is:

>  Don't show followers (they are) if the follower use Odoo messaging without emails

I think it would be a good time to fix this problem too, if you want.

The point is that you should include in `recipients` also those that are notified by other systems (not only mail; also those that read notifications in Odoo directly).

If you don't feel like so, this can be handled later, but at least it would be nice if you can add it to `readme/ROADMAP.rst` in case it isn't there yet.